cafebabel/static/css/styles.css

```diff
@@ -2256,6 +2256,61 @@ a.section-link.creative {
 
 /* Footer */
 
+#static-pages-links {
+  margin-bottom: 70rem;
+  background-color: #00283c;
+  height: 10rem;
+  padding: 2.2rem;
+}
+
+#static-pages-links a {
+  color: #fff;
+}
+
+#static-pages-links a:after {
+  background-color: #fff;
+}
+
+#static-pages-links ul {
+  margin: 0 auto;
+  max-width: 75rem;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 1rem;
+}
+
+#static-pages-links ul li {
+  display: inline-block;
+  font-size: 1.3rem;
+  font-weight: 100;
+}
+
+#static-pages-links ul li ul,
+#static-pages-links ul li ul li {
+  display: block;
+}
+
+#static-pages-links ul li ul li {
+  padding: 0.5rem 0;
+}
+
+#static-pages-links ul li ul li ul,
+#static-pages-links ul li ul li ul li {
+  display: inline-block;
+}
+
+#static-pages-links ul li ul li ul li {
```

Comment:
I don't think you need all the intermediaries, like `ul ul ul li` is enough :)

Reply:
Same for others of course ;)

Reply:
Sure :-)